src/test/projects/processors/pom.xml

```diff
@@ -0,0 +1,16 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>fr.inria.gforge.spoon</groupId>
+  <artifactId>processors</artifactId>
+  <version>1.0-SNAPSHOT</version>
+	<dependencies>
+	<dependency>
+		<groupId>fr.inria.gforge.spoon</groupId>
+		<artifactId>spoon-core</artifactId>
+		<version>8.1.0</version>
```

Comment:
Would be better on long term view to make it configurable or at least based on the version specified in the actual pom.xml of the project.

Reply:
this is only example code plus it only depends on core stable features of spoon (extends Processor), so the overhead maintenance in the build script is not worth it IMHO. OK for you?

Reply:
Ah indeed it will be only used to compile the processor to be used in the test, right? Should be good indeed then.